### PR TITLE
Ensure gocv.Mat(s) returned by ForwardLayers are Closed()

### PR DESCRIPTION
**Pull request by arwineap**
This should allow the memory to be properly released

### CODE REVIEW

**Comment by wimspaargaren** [comment]
Hello @arwineap !

Thx for your contribution :) Do you think it makes sense to actually check the `Mat.Close` error? If not, we might want to ignore it all together, like [how the image blob is closed](https://github.com/wimspaargaren/yolov3/pull/8/files#diff-fa59cb8e201ce161b0d301eca54a71ea0248bd8835b9e5071a2856838fdf4a6aR175-R176).

File: yolov3.go [hunk 1]
```diff
@@ -192,13 +196,12 @@ func (y *yoloNet) processOutputs(frame gocv.Mat, outputs []gocv.Mat, filter map[
...
 	for i := 0; i < len(outputs); i++ {
-		output := outputs[i]
-		data, err := output.DataPtrFloat32()
+		data, err := outputs[i].DataPtrFloat32()
 		if err != nil {
 			return nil, err
 		}
-		for i := 0; i < output.Total(); i += output.Cols() {
-			row := data[i : i+output.Cols()]
+		for x := 0; x < outputs[i].Total(); x += outputs[i].Cols() {
```
Comment (wimspaargaren) [inline yolov3.go:203]: Should we use `j` instead of `x` here?

**Comment by arwineap** [comment]
Yes, it would be better to defer the Close statements just in case processOutputs returns an error :+1: 

Thanks for the code, as well as the review

File: yolov3.go (hunk 1 above)
Comment (arwineap) [reply yolov3.go]: yes, that makes sense 👍

**Review by wimspaargaren — APPROVED** [review]

**Pull request merged**